tests/test_cli.py

```diff
@@ -73,12 +73,13 @@ def run_successful(self, result: Result, test_case: SSSOMTestCase) -> None:
 
     def run_convert(self, runner: CliRunner, test_case: SSSOMTestCase) -> Result:
         """Run the convert test."""
+        fmt = "owl"
```

Comment:
relevant?

Reply:
no ... I usually don't add irrelevant changes but this was minor, so thought of just pushing it.

Reply:
kk :shipit: